### PR TITLE
Nc news 11

### PR DESCRIPTION
**Pull request by SandorGonczi**
.only removed , tests are passing

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -292,4 +292,103 @@ describe("/api/articles/:article_id/comments", () => {
...
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        expect(articles).toEqual(expect.any(Array));
+        expect(articles).toHaveLength(0);
+      });
+  });
+  test("GET:404 error message when invalid order sort by value passsed", () => {
+    return request(app)
+      .get("/api/articles/?topic=paper&sortBy=droptable--")
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:346]: nice 😆

File: __tests__/app.test.js [hunk 2]
```diff
@@ -292,4 +292,103 @@ describe("/api/articles/:article_id/comments", () => {
...
+  });
+  test("GET:404 error message when invalid query parameters passsed", () => {
+    return request(app)
+      .get("/api/articles/?topic=nontopic")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("No such topic exist!");
+      });
+  });
+  test("GET:404 error message when misspelled query key passed query parameters passsed", () => {
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:368]: Invalid or bad requests should be a 400 status.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -292,4 +292,103 @@ describe("/api/articles/:article_id/comments", () => {
...
+    return request(app)
+      .get("/api/articles/?topi=mitch")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Invalid Request!");
+      });
+  });
+  test("GET:404 error message when misspelled query key passed query parameters passsed", () => {
+    return request(app)
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:377]: Invalid or bad requests should be a 400 status.

File: __tests__/app.test.js [hunk 4]
```diff
@@ -292,4 +292,103 @@ describe("/api/articles/:article_id/comments", () => {
...
+      .get("/api/articles/?topic=mitch&orde=asc")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Invalid Request!");
+      });
+  });
+  test("GET:404 error message when misspelled query key passed query parameters passsed", () => {
+    return request(app)
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:385]: Invalid or bad requests should be a 400 status.

**Review by dappernerddesigns — APPROVED** [review]
There are some status codes to change here. Please amend the invalid or bad request tests to 400 and make sure they all pass before moving on.

File: __tests__/app.test.js (hunk 4 above)
Comment (SandorGonczi) [reply __tests__/app.test.js:385]: fixing it before merge

**Pull request closed without merging**